server/routes/entryRoute.js

```diff
@@ -1,14 +1,15 @@
 import express from 'express';
 import entry from '../controllers/entries';
 import validate from '../middlewares/validate';
+import auth from '../helpers/authentication';
 
 const router = express.Router();
 
-router.get('/', entry.getAll);
-router.get('/:entryId', validate.entryId, entry.getOne);
-router.post('/', validate.createEntry, entry.createEntry);
-router.put('/:entryId', validate.entryId, validate.modifyEntry, entry.modifyEntry);
-router.delete('/:entryId', validate.entryId, entry.deleteEntry);
+// router.get('/', entry.getAll);
```

Comment:
please remove commented code if you're no longer using it

Reply:
I will remove them